mmf/trainers/core/evaluation_loop.py

```diff
@@ -44,6 +46,8 @@ def evaluation_loop(
 
             combined_report.metrics = self.metrics(combined_report, combined_report)
             self.update_meter(combined_report, meter, eval_mode=True)
+            logger.info("Validation Done")
+            metsumm("After Validation Complete")
```

Comment:
this should go away, it's a debugging artifact. should not live in mmf codebase.